src/components/Person/Person.jsx

```diff
@@ -1 +1,16 @@
-// export const Person = ({ person }) => ();
+const MALE = 'm';
+const partner = sex => (sex === MALE ? 'wife' : 'husband');
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age && <p className="Person__age">{`I am ${person.age}`}</p>}
+    {person.isMarried ? (
+      <p className="Person__partner">
+        {`${person.partnerName} is my ${partner(person.sex)}`}
+      </p>
+    ) : (
+      <p className="Person__partner">I am not married </p>
```

Comment:
We are repeating <p className="Person__partner"> 2 times . Let's do the check inside a p tag so that will have it only once and I would move it to a variable for better readability

```
const someVar = check here
<p>{someVar}</p>
```